extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -159,6 +240,48 @@ class AmpLightbox extends AMP.BaseElement {
     this.active_ = true;
   }
 
+  /** @private */
+  maybeEnterFrameFullOverlayMode_() {
+    if (this.isInMainDocument_()) {
+      return;
+    }
+
+    if (!isExperimentOn(this.getAmpDoc().win, A4A_PROTOTYPE_EXPERIMENT)) {
```

Comment:
move this up before the isInMainDocument to make the experiment completely side-effect free.